src/status_im/contexts/wallet/data_store.cljs

```diff
@@ -242,35 +243,34 @@
-                                                                                        new-path)
-                                                                                       precision)
+     :gas-fees                  {:gas-price "0"
+                                 :base-fee (send-utils/convert-to-gwei (:tx-base-fee
+                                                                        new-path)
+                                                                       precision)
                                  :max-priority-fee-per-gas (send-utils/convert-to-gwei (:tx-priority-fee
                                                                                         new-path)
                                                                                        precision)
-                                 :max-fee-per-gas-low      (send-utils/convert-to-gwei
-                                                            (get-in
-                                                             new-path
-                                                             [:suggested-levels-for-max-fees-per-gas
-                                                              :low])
-                                                            precision)
-                                 :max-fee-per-gas-medium   (send-utils/convert-to-gwei
-                                                            (get-in
-                                                             new-path
-                                                             [:suggested-levels-for-max-fees-per-gas
-                                                              :medium])
-                                                            precision)
-                                 :max-fee-per-gas-high     (send-utils/convert-to-gwei
-                                                            (get-in
-                                                             new-path
-                                                             [:suggested-levels-for-max-fees-per-gas
-                                                              :high])
-                                                            precision)
-                                 :l-1-gas-fee              (send-utils/convert-to-gwei (:tx-l-1-fee
-                                                                                        new-path)
-                                                                                       precision)
-                                 :eip-1559-enabled         true}
+                                 :l-1-gas-fee (send-utils/convert-to-gwei (:tx-l-1-fee
+                                                                           new-path)
+                                                                          precision)
+                                 :eip-1559-enabled true
+                                 :tx-max-fees-per-gas (send-utils/convert-to-gwei
```

Comment:
This is a new field from `status-go` that holds currently used gas. It need to be used for fee calculation